test/common/common.go

```diff
@@ -258,3 +258,11 @@ func DoRootComplianceTest(hub dynamic.Interface, policyName string, compliance p
 		1,
 	).Should(gomega.Equal(compliance))
 }
+
+func OutputDebugInfo(testName string) {
+	ginkgo.GinkgoWriter.Printf("%s test Kubernetes info:\n", testName)
+
+	for _, resource := range []string{"policies", "placementrule", "placement", "placementbinding"} {
```

Comment:
I think we should qualify the "policies" name - do you want kyverno policies, open-cluster-management policies, maybe both?